tsl/src/compression/create.c

```diff
@@ -736,18 +753,23 @@ validate_existing_constraints(Hypertable *ht, CompressColInfo *colinfo)
 			attnums = (int16 *) ARR_DATA_PTR(arr);
 			for (j = 0; j < numkeys; j++)
 			{
-				int16 colno = attnums[j] - 1;
+				FormData_hypertable_compression *col_def =
```

Comment:
if the column was dropped, why do we still see the constraint on the table? Is the constraint marked as not valid?

Reply:
we don't but the index to col_meta will not match the attnum because col_meta does not include dropped columns